tests/basic.test.tsx

```diff
@@ -177,11 +177,14 @@ it('only re-renders if value has changed', async () => {
   type Props = { countAtom: typeof count1Atom; name: string }
   const Counter: React.FC<Props> = ({ countAtom, name }) => {
     const [count, setCount] = useAtom(countAtom)
-    const renderCount = React.useRef(0)
+    const commits = useRef(1)
+    useEffect(() => {
+      ++commits.current
+    })
     return (
       <>
         <div>
-          renderCount: {++renderCount.current}, {name}: {count}
```

Comment:
Because use-context-selector v1.2 useContextUpdate forces double renders for better CM support, we can't rely on `renderCount` for testing. So, we use commits count instead.